### PR TITLE
Add vision that doesn't suck a lot

### PR DESCRIPTION
**Pull request by ritikmishra**
It only sucks a little bit.

### CODE REVIEW

**Review by isaacaj — APPROVED** [review]
Looks good. I might refactor some things in a minute and will then move on to testing.

File: src/main/java/com/team2502/robot2019/OI.java [hunk 1]
```diff
@@ -68,6 +72,7 @@
 
     public static final Button BUTTON_SWITCH_DIRECTION = new JoystickButton(JOYSTICK_DRIVE_RIGHT, RobotMap.Joystick.Button.BUTTON_SWITCH_DIRECTION);
 
+    public static final Button BUTTON_APPLE = new JoystickButton(JOYSTICK_DRIVE_LEFT, 8);
```
Comment (ryandalex1) [inline src/main/java/com/team2502/robot2019/OI.java:75]: Tsk tsk Ritik

File: src/main/java/com/team2502/robot2019/OI.java [hunk 2]
```diff
@@ -87,6 +95,7 @@
         BUTTON_RUN_CARGO_ACTIVE_BWD_BOTTOM.whileHeld(new CargoActiveCommand(CargoSubsystem.Belt.BOTTOM, Constants.Physical.CargoActive.SPEED_BKWDS));
 
 
+        BUTTON_APPLE.whileHeld(new VelocityDriveCommand(4, 4, 1));
```
Comment (ryandalex1) [inline src/main/java/com/team2502/robot2019/OI.java:98]: Still Needed?

File: src/main/java/com/team2502/robot2019/command/vision/AlwaysListeningCommand.java [hunk 3]
```diff
@@ -21,7 +26,7 @@ protected void initialize() {
 
 
         try {
-            socket = new VisionWebsocket("team2502-tinker.local", 5800);
+            socket = new VisionWebsocket(Constants.Autonomous.COPROCESSOR_MDNS_ADDR, 5800);
```
Comment (ryandalex1) [inline src/main/java/com/team2502/robot2019/command/vision/AlwaysListeningCommand.java:29]: Should 5800 be a constant

**Review by ryandalex1 — APPROVED** [review]

File: src/main/java/com/team2502/robot2019/command/vision/AlwaysListeningCommand.java (hunk 3 above)
Comment (ritikmishra) [reply src/main/java/com/team2502/robot2019/command/vision/AlwaysListeningCommand.java:29]: Yes

File: src/main/java/com/team2502/robot2019/OI.java (hunk 2 above)
Comment (ritikmishra) [reply src/main/java/com/team2502/robot2019/OI.java:98]: No

File: src/main/java/com/team2502/robot2019/OI.java (hunk 1 above)
Comment (ritikmishra) [reply src/main/java/com/team2502/robot2019/OI.java:75]: It is no longer needed

**Pull request merged**